api/compute/v1alpha1/common.go

```diff
@@ -390,21 +390,24 @@ const (
 )
 
 func validResourceRequirement(requirements corev1.ResourceRequirements) bool {
+	if requirements.Limits.Cpu().Sign() == 0 {
```

Comment:
Leave `cpu` to empty will make it use the default "zero" value, but different from passing a 0 value explicitly, it won't set the CPU resources